exercises/concept/city-office/test/form_test.exs

```diff
@@ -85,15 +85,25 @@ defmodule FormTest do
           declaration == :type && type_name == unquote(type_name)
         end)
 
+      assert type,
+             "expected the module #{unquote(module_name)} to have a public type named #{unquote(type_name)}"
+
```

Comment:
This is an unrelated bug fix. This assertion was too late, causing a cryptic `no match of right hand side value: nil` error when a type was missing.